### PR TITLE
83 integration all forms camilo

### PR DESCRIPTION
**Pull request by soycamis**
The code is commented, because you still have to create the templates for each form as they change the fields.

### CODE REVIEW

File: src/js/careers.forms.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  invalidHandler: function(event, validator) {
+    var errors = validator.numberOfInvalids();
+    alertify.logPosition("bottom right");
+    alertify.error("Verifica tu información tienes " + errors + " errores.");
+  }
+});
+
+$('form[name="form-careers-2"]').validate({
+  // required fields
+  LiCareers: 'required',
```
Comment (Enrique199413) [inline src/js/careers.forms.js:39]: what does mean `Li` prefix

File: src/js/quote.form.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    // Specify that email should be validated
+    // by the built-in "email" rule
+    email: true
+  },
+  messages: {
+    emailQuote: "Por favor introduce un correo valido."
+  },
+  submitHandler: function(form) {
+    var data = $('form[name="form-quote"]').serialize();
+    console.log(data);
```
Comment (Enrique199413) [inline src/js/quote.form.js]: Remove console.log and if you dont use the AJAX code remove it

File: src/js/referrals.form.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    // Specify that email should be validated
+    // by the built-in "email" rule
+    email: true
+  },
+  messages: {
+    emailCareers: "Por favor introduce un correo valido."
+  },
+  submitHandler: function(form) {
+    var data = $('form[name="form-referrals"]').serialize();
+    // $.ajax({
```
Comment (Enrique199413) [inline src/js/referrals.form.js]: If you dont use this part of code, remove it

**Review by Enrique199413 — CHANGES_REQUESTED** [review]

File: src/js/quote.form.js (hunk 2 above)
Comment (soycamis) [reply src/js/quote.form.js]: removed

File: src/js/careers.forms.js (hunk 1 above)
Comment (soycamis) [reply src/js/careers.forms.js:39]: amm that did it lil :D ... this is the attribute name in the form of "Li"nkedin...

File: src/js/referrals.form.js (hunk 3 above)
Comment (soycamis) [reply src/js/referrals.form.js]: will be used once the views in integration

**Review by Enrique199413 — APPROVED** [review]

**Pull request merged**